rosbag2_examples/rosbag2_examples_py/rosbag2_examples_py/simple_bag_recorder.py

```diff
@@ -41,7 +41,6 @@ def __init__(self):
             'chatter',
             self.topic_callback,
             10)
-        self.subscription
```

Comment:
@MichaelOrlov can you take a look at this? typo?

Reply:
Ah, okay.

i do not see any warnings with `colcon test --event-handlers console_direct+ --packages-select examples_rclpy_minimal_subscriber` or `colcon test --event-handlers console_direct+ --packages-select rosbag2_examples_py` even if i remove this, but some other platform can detect it or maybe missing something else... anyway no big deal, i will go ahead to close this. thanks 👍 for the comment!

Reply:
@fujitatomoya Sorry, I don't know, since I am not an author of this code. Maybe @emersonknapp can recall since as far as I remember he was an author of it in the rosbag2_examples.